docs/configuration/settings.rst

```diff
@@ -5,113 +5,469 @@ Mautic Configuration Settings
+
+* **Unsubscribed and resubscribed confirmation message** - If a Contact unsubscribes or resubscribes, this message displays on the page after the respective action. Don’t edit the ``|EMAIL|`` or the ``|URL|`` token in the ``<a href>`` tag.
+
+* **Show Contact preference settings** - Select **Yes** to direct the unsubscribe link to your configured preference center. If you haven’t created a preference center, Mautic creates a default page based on the next 5 settings. The created page uses the default Theme for styling.
+
+* **Show Contact Segment preferences** - Select **Yes** to enable a Contact to change which Segments they’re part of on the preference center page. Segments won’t display on the preference center page if they aren’t published and public.
+
+* **Show Contact frequency preferences** - Select **Yes** to enable an individual to limit the number of marketing messages they receive on each Channel from the preference center.
+
+* **Show pause Contact preferences** - Select **Yes** to enable Contacts to turn-off messages from your Mautic account to their Email address for a specified date range. This action isn’t a full unsubscribe, and at the end of the date range, the message is sent to that address once again.
+
+* **Show Contact’s Categories** - If you have Categories set for Contacts, Campaigns, Emails, etc., select Yes to enable the Contact to opt out of the Categories they choose from the preference center page.
+
+* **Show Contact’s preferred Channel option** - If you have multiple Channels available within your Mautic account (Email, SMS, mobile push, web notifications, etc.), Contacts can choose their preferred Channel. This can be useful if you are using the Marketing Messages feature of Mautic. More information about the Preference Center is available :doc:`here</contacts/preference_center>`.
+
+
 Form settings
 *************
 
+.. image:: images/form-settings.png
+  :width: 600
+  :alt: Screenshot showing Form Settings Configuration in Mautic
+
+* **Do not accept submission from these domain names** - To block Contacts with specific Email domains from submitting your Forms, enter those domains in the dialog box. Select an option on each Form you want to apply this block to. You can restrict either specific Email aliases that belong to a domain or an entire domain. To block the entire domain, you can use wildcards (*).
+
 Contact settings
 ****************
 
 Contact merge settings
 ======================
 
+.. image:: images/contact-merge-settings.png
+  :width: 600
+  :alt: Screenshot showing Contact Merge Settings Configuration in Mautic
+
+* **Merge by unique fields with operator** - If you use more than one unique fields identifiers then you can define the way of the find to merge it. 
+
 Contact list settings
 =====================
 
+.. image:: images/contact-list-settings.png
+  :width: 600
+  :alt: Screenshot showing Contact List Settings Configuration in Mautic
+
+* **Columns** -
```

Comment:
Is this waiting for text? Seems like an incomplete section.

Reply:
Yes, looking for the best and simplest way to describe that section. Can you help with this? @RCheesley

Reply:
Sure, this section allows you to choose from the left, which fields are shown on the contact lists (when you go to contacts in the front-end of Mautic and view the list).

So they have to select the fields from the left and move them to the right to show them in the list. Or remove from the right if they don't want them to be shown.